libs/solid/profile/src/lib/components/detail/detail.component.html

```diff
@@ -218,7 +218,7 @@
     <ng-container *ngSwitchCase="PropertyTypes.String">
       <p
         *ngIf="
-          !(profile_obj[property.key].indexOf('http') === 0);
+          !(profile_obj[property.key]?.indexOf('http') === 0);
```

Comment:
`!` und `?` in einem command... that's programmer's hell 😈

Reply:
Ah... es ist ja ein NOT und kein Bang Nagut... da war ich zu schnell und bin still 😉